README.md

```diff
@@ -27,6 +27,8 @@ The bot can be found at https://t.me/GamersSocialManagerBot
 /about -- The bot will reply with general information about itself.
 
 /start -- Used to start up the bot.
+
+/diss [username with or without @] -- Used to issue a random diss to the mentioned user.
```

Comment:
If you're doing this, it might be nice for you to add this to the /help command as well (probably best to add in the "memes" section of the help command).

Reply:
<img width="713" alt="Screen Shot 2022-10-20 at 8 48 33 PM" src="https://user-images.githubusercontent.com/28017034/197107306-0dc20009-9e92-49fc-a618-811638704cf8.png">